rpc/js/src/BaseChannel.ts

```diff
@@ -12,7 +12,7 @@ export class BaseChannel {
     private closed = false;
     private closedReason?: Error;
 
-    protected maxDataChannelSize = 16384;
+    protected maxDataChannelSize = 65535;
```

Comment:
[q] why are we changing this if it is a no op?

Reply:
consistency!